src/screens/Customers/components/CustomerCard.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { createStyles, Theme, withStyles } from '@material-ui/core/styles';
+import { Card, CardActions, Typography, IconButton } from '@material-ui/core';
+import ArrowForwardIosIcon from '@material-ui/icons/ArrowForwardIos';
+import AttachMoneyIcon from '@material-ui/icons/AttachMoney';
+import FlashOnIcon from '@material-ui/icons/FlashOn';
+
+const styles = (theme: Theme) =>
+  createStyles({
+    arrow: {
+      color: theme.palette.text.secondary,
+      fontSize: '15px',
+      marginLeft: '0px',
+    },
+    cardContent: {
+      flexGrow: 1,
+      display: 'flex',
+      alignItems: 'center',
+      paddingLeft: '17px',
+    },
+    totalText: {
+      color: theme.palette.text.secondary,
+    },
+    titleText: {
+      fontSize: '19px',
+    },
```

Comment:
Don't use specify fontsize. typography `h2` should already adjust font size for you appropriately